### ISSUE TITLE
Release 2.0.0 - Missing dependencies? Read this

### ISSUE THREAD

**Issue by bblommers** [spulec/moto#3722]
## New Release

Moto 2.0.0 has just been released - see our [CHANGELOG](https://github.com/spulec/moto/blob/master/CHANGELOG.md) for a full overview of all changes.

#### Note that the installation method has changed since 1.x.
To install moto for a specific service:

```
pip install moto[ec2,s3]
```

This will install Moto, and only the dependencies required for that specific service.
If you don't care about the number of dependencies, or if you want to mock many AWS services:

```
pip install moto[all]
```

Not all services might be covered, in which case you might see a warning:
_moto 1.3.16 does not provide the extra 'service'._
You can ignore the warning, or simply install moto as is:
```
pip install moto
```

**Comment by jacobsorme** [comment]
I seem to be getting errors, `moto/awslambda/models.py", line 10, in <module>
    import docker
ModuleNotFoundError: No module named 'docker'`.  Before I had no problems with my dependencies. Is this related to 2.0.0? Thanks.

**Comment by amarjandu** [comment]
@bblommers is version 1.3.17 going to be released? Or did 2.0.0 take over? I tried searching around, nor can find any tags mentioning it.

**Comment by bblommers** [comment]
@amarjandu No, there will be no 1.3.17 release. We've moved to 2.x because of this breaking change
